frontend/javascripts/oxalis/view/left-border-tabs/histogram_view.js

```diff
@@ -40,7 +48,7 @@ type HistogramState = {
 
 const uint24Colors = [[255, 65, 54], [46, 204, 64], [24, 144, 255]];
 const canvasHeight = 100;
-const canvasWidth = 300;
+const canvasWidth = 318;
```

Comment:
Can you explain the reason for this change?

Reply:
I just noticed that the right edges weren't aligned at 300 pixels (which felt like an arbitrary number).
![Screenshot 2022-01-18 at 20 39 01](https://user-images.githubusercontent.com/46814136/150006922-04b7864d-4cc8-4bff-9678-3b4b975bbb27.png)

Reply:
Ah, cool, thanks for fixing this :) :100: